packages/big-design/src/components/Modal/__snapshots__/spec.tsx.snap

```diff
@@ -314,7 +314,31 @@ exports[`render open modal without backdrop 1`] = `
 <body
   style="overflow-y: hidden;"
 >
-  <div />
+  @media (min-width:720px) {
+
+}
+
+@media (min-width:720px) {
```

Comment:
These look a bit odd 🤔

Reply:
I noticed that too... My thought is that we are including media queries based on prop values inside `css`:
```tsx
  ${({ theme, variant }) =>
    variant === 'modal' &&
    css`
      ${theme.breakpoints.tablet} {
        /* ... */
      }

      ${theme.breakpoints.desktop} {
        /* ... */
      }
    `}
```

Let me do some more digging on this though

Reply:
So I dove a bit deeper into why this was happening, and it seems to be a [known issue](https://github.com/styled-components/jest-styled-components/issues/276) in `jest-styled-components`. It's a pretty gnarly bug on their end and after looking at their code it seems like each test is overwriting a reference somewhere. When I run this test with `.only` the snapshots look fine.

There are two courses of action for this:
1. See if we can figure out the issue in the lib and submit a fix.
2. Ignore this for now, and monitor to see if they fix it... though it's been 2 years since the issue was first submitted.

After chatting with @deini I think we'll ignore this and monitor for a fix.